WordPressKit/WordPressKit/AccountServiceRemoteREST.m

```diff
@@ -194,11 +191,11 @@ - (void)requestWPComAuthLinkForEmail:(NSString *)email success:(void (^)())succe
 
     NSMutableDictionary *params = [NSMutableDictionary dictionaryWithDictionary:@{
                                                                                   @"email": email,
-                                                                                  @"client_id": [ApiCredentials client],
-                                                                                  @"client_secret": [ApiCredentials secret],
+                                                                                  @"client_id": clientID,
+                                                                                  @"client_secret": clientSecret,
                                                                                   }];
-    if (![@"wordpress" isEqualToString:WPComScheme]) {
-        [params setObject:WPComScheme forKey:@"scheme"];
+    if (![@"wordpress" isEqualToString:scheme]) {
```

Comment:
Definitely something we can address outside of this PR. Since I'm moving all existing code I want to try to change as little as practical (even though I'm doing some heavy refactoring).